src/main/resources/fileTemplates/config.jelly.ft

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright DATE, Issuer.....
+
+Licensed under ...
+
+@Author
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" >
+    <f:section title="${%Put Section Name Here}">
```

Comment:
why are we adding a section by default, lots of config pages won't need that, e.g. if they are included in another one

Reply:
The guide suggests creating a template based on a file and then exporting the result and adding to a plugin. I'm guessing this template is based on config view with a section.

Are there multiple type of views each with different boilerplate?

What's the current process for creating a view? What would ideal one look like?

Reply:
just an empty jelly file I think with the form and layout namespaces imported as they are the most common ones

Reply:
This template is based on config view with a section. Is the plan to have several different templates? In which case we can have ones with sections and ones without.

Reply:
based off of https://plugins.jetbrains.com/docs/intellij/providing-file-templates.html#file-templates-categories I don't think it should include section, just an empty jelly file should be fine

Reply:
OK. Let us remove the section and keep it bare-bones jelly that can be used for any view or tagfile.
What should we name it (i.e., filename, which is used as a template name in IntelliJ UI)? "Stapler Jelly View"?

Reply:
I don’t think stapler should be in its name.

Jelly file
Jelly view

would both be fine I think

Reply:
Well... `/lib/form` namespace is a Jenkins thing. Jelly is not popular but still has some use elsewhere. 
[ServiceNow Jelly tags](https://docs.servicenow.com/en-US/bundle/tokyo-application-development/page/script/general-scripting/reference/r_JellyTags.html) < ServiceNow seems to be using it. Hence, I suggested an extra qualifier in the name. Maybe it should be "Jenkins Jelly View", not Stapler. 

Maybe I am overthinking. ServiceNow seems to have a built-in editor, so it might be unlikely that any SNOW developer will discover this plugin because IntelliJ says that it supports `*.jelly` extension. Even if they do, they'll probably see the plugin name and avoid installing it anyway. 🤷

Reply:
Agreed

Reply:
@julieheard please do the above two actions and I will merge the PR. Thank you!

Reply:
Both changes done